ofdb-core/src/usecases/tests.rs

```diff
@@ -262,6 +262,12 @@ impl PlaceRepo for MockDb {
     ) -> RepoResult<Vec<(Place, ReviewStatus, ActivityLog)>> {
         unimplemented!();
     }
+    fn find_places_not_updated_since(
+        &self,
+        _not_updated_since: Timestamp,
+    ) -> RepoResult<Vec<(Place, ReviewStatus)>> {
+        unimplemented!();
```

Comment:
`unimplemented!("FIXME: optional text");`

Reply:
I have no idea what text I should use that adds any value.